src/git/blame.ts

```diff
@@ -374,6 +381,46 @@ export class GitBlame {
         }
     }
 
+    private async getRemoteUrl(): Promise<string> {
+        if (
+            !isActiveEditorValid()
+            || !(
+                window
+                && window.activeTextEditor
+            )
+        ) {
+            return "";
+        }
+        const gitCommand = getGitCommand();
+        const activeFile = window.activeTextEditor.document.fileName;
+        const activeFileFolder = parse(activeFile).dir;
+        const currentBranch = await execute(gitCommand, [
+            "symbolic-ref",
+            "-q",
+            "--short",
+            "HEAD",
+        ], {
+            cwd: activeFileFolder,
+        });
+        const curRemote = await execute(gitCommand, [
+            "config",
+            "--worktree",
+            "--get",
+            `branch.${ currentBranch.trim() }.remote`,
+        ], {
+            cwd: activeFileFolder,
+        });
+        const remoteUrl = await execute(gitCommand, [
+            "config",
+            "--worktree",
```

Comment:
I use git for windows version `2.18.0.windows.1` to do my local testing and I do not have a `--worktree` parameter. I can not find it [in the documentation](https://git-scm.com/docs/git-config). Are you able to clarify what the intent of the parameter is or if it can be removed?

Reply:
TIL that my git is 2.20.

Documentation is here: https://github.com/git/git/blob/master/Documentation/git-config.txt#L127

It's merely a flag that works just like --local (use the config in .git/config) but you can configure it a bit. Replaced with --local